android_splash_screen/android/app/src/main/kotlin/com/example/splash_screen_sample/MainActivity.kt

```diff
@@ -0,0 +1,202 @@
+package com.example.splash_screen_sample
```

Comment:
Since you've modified this file, it needs a copyright header. Just copy the one used elsewhere in the repo.